src/test/java/uk/gov/companieshouse/overseasentitiesapi/utils/TestUtils.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.logging.Logger;
+
+/**
+ *  Utilities to read test resources.
+ */
+public final class TestUtils {
+
+    private static final Logger LOGGER = Logger.getLogger(TestUtils.class.getName());
+
+    public String readFileReturnString(File filename) {
```

Comment:
rename to readString(filename)? Java 11 has File.readString()

Reply:
Yes, agree (copied this code from somewhere else so should have changed that method name).